src/index.js

```diff
@@ -452,27 +458,40 @@ class LocalstackPlugin {
       return containerID;
     }
 
+    const startContainer = () => {
+      this.log('Starting LocalStack in Docker. This can take a while.');
+      const cwd = process.cwd();
+      const env = this.clone(process.env);
+      env.DEBUG = '1';
+      env.LAMBDA_EXECUTOR = env.LAMBDA_EXECUTOR || 'docker';
+      env.LAMBDA_REMOTE_DOCKER = env.LAMBDA_REMOTE_DOCKER || '0';
+      env.DOCKER_FLAGS = (env.DOCKER_FLAGS || '') + ` -v ${cwd}:${cwd}`;
+      env.START_WEB = env.START_WEB || '0';
+      const maxBuffer = (+env.EXEC_MAXBUFFER)||50*1000*1000; // 50mb buffer to handle output
+      if (this.shouldRunDockerSudo()) {
+        env.DOCKER_CMD = 'sudo docker';
+      }
+      const options = {env: env, maxBuffer};
+      return exec('localstack start', options).then(getContainer)
```

Comment:
I tested this again, and now the `sls deploy --stage local` process is not properly closing. 
(Previously I had the issue that the container wouldn't start at all, because the `-d` was parsed as a docker flag, but is not recognized for the `create-container` command.)

I think we need the d-flag with the start command though, e.g. 
`exec('localstack start -d', options).then(getContainer)`

When I run the deploy with this options localstack community image starts up  and the process finishes :)